views/Admin.react.js

```diff
@@ -739,6 +745,30 @@ class AdminPage extends Component {
                 dataKey="access"
                 width={this.state.width / 6}
               />
+              <Column
+                label="Account Expiration"
+                cellRenderer={({ cellData, rowData: { uid } }) => (
+                  console.log({ cellData, uid }),
+                  (
+                    <TextField
+                      id="account_expiration"
+                      type="date"
+                      defaultValue={moment(cellData).format('YYYY-MM-DD')}
```

Comment:
should we handle there not being an account_expires value?

Reply:
The field automatically handles it by displaying the date format of mm-dd-yyyy when there is no value